.eslintignore

```diff
@@ -3,6 +3,7 @@ lib/punycode.js
 test/addons/??_*/
 test/fixtures
 test/tmp*/
+test/es-module/test-esm-dynamic-import.js
```

Comment:
Afaict our ESLint doesn't support dynamic import (and won't while it's still in stage 3).